src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -63,6 +63,7 @@
 #include "utilities/align.hpp"
 #include "utilities/dtrace.hpp"
 #include "utilities/events.hpp"
+#include "utilities/globalCounter.inline.hpp"
```

Comment:
Why is this necessary?

Reply:
This compilation unit uses the `OM_PERFDATA_OP` define, which now uses `GlobalCounter::CriticalSection`, which now needs this include.